Dockerfile.build

```diff
@@ -2,9 +2,20 @@ FROM archlinux:base-devel
 
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
-    PATH=/usr/local/cargo/bin:$PATH
+    PATH=/usr/local/cargo/bin:$PATH \
+    POETRY_HOME=/opt/poetry \
+    CLIENT_CLI_DIR=/__w/iroha/iroha/target/debug
 
-RUN pacman -Syu rustup mold musl rust-musl openssl libgit2 git docker docker-buildx docker-compose --noconfirm
+ENV PATH=$POETRY_HOME/bin:$PATH
```

Comment:
Two env vars with the same names? `PATH`

Reply:
> Two env vars with the same names? `PATH`

I broken it into two steps for better readability. IMHO